bloom/infra/repositories/file_repository_vessel.py

```diff
@@ -0,0 +1,44 @@
+import json
+from csv import DictWriter
+from logging import getLogger
+from pathlib import Path
+
+import pandas as pd
+
+from bloom.domain.vessel import AbstractVessel, Vessel, VesselPositionMarineTraffic
+
+logger = getLogger()
+
+
+class VesselRepository(AbstractVessel):
+    def __init__(self) -> None:
+        self.vessels_path = Path.joinpath(Path.cwd(), "data/chalutiers_pelagiques.xlsx")
+        self.results_path = Path.joinpath(Path.cwd(), "data/bloom_scrap.csv")
+
+    def load_vessel_identifiers(self) -> list[Vessel]:
+        df = pd.read_excel(self.vessels_path, engine="openpyxl")
+        vessel_identifiers_list = df["IMO"].tolist()
+
+        return [Vessel(IMO=imo) for imo in vessel_identifiers_list]
+
+    def save_vessels_positions(
```

Comment:
why do we need to save vessel position in a file ? and load identifiers with python (an sql script would be anough, no?) ?

Reply:
to rm